src/App.css

```diff
@@ -39,134 +32,297 @@ a {
 
 .Nav-parent-container {
   display: flex;
-  margin: 2% auto 0;
-  width: 50%;
+  justify-content: flex-start;
+  background-color: white;
+  padding: 1rem 2rem;
+  border-bottom: 1px solid #ccc;
+  gap: 2rem;
+  padding-left: 7.5%;
   align-items: center;
-  justify-content: space-between;
+  width: 100%;
+  margin: 0 auto;
 }
 
 .Nav-parent-container h1 {
-  width: 50%;
+  color: #0290ff;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.8rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
 }
 
 .nav-container {
   font-size: large;
   width: 30%;
   display: flex;
   align-items: center;
-  justify-content: flex-end;
   gap: 10%;
-  margin: auto;
+  margin: 0;
+  padding: 0;
+  opacity: 0.6;
 }
 
 .nav-container li:hover {
   text-decoration: underline;
+  font-weight: bolder;
+}
+
+.nav-container li:focus {
+  color: #030000;
+  font-weight: bolder;
+}
+
+.nav-container li:active {
+  font-weight: bolder;
+}
+
+.link-active {
+  text-decoration: underline;
+  color: black;
+  font-weight: 800;
+}
+
+.user_icon {
+  color: #0290ff;
+  display: inline-flex;
+  align-items: center;
+  margin-left: auto;
+  margin-right: 4rem;
+}
+
+.fa-icon {
+  width: 4rem;
+  height: 2rem;
 }
 
 .books-container {
   display: flex;
   flex-direction: column;
   margin: 0 auto;
-  width: 90%;
+  width: 85%;
   font-size: larger;
+  padding: 0;
 }
 
-.book-buttons {
+.book {
+  background-color: white;
   display: flex;
-  justify-content: center;
-  gap: 5%;
-  margin: 0 auto;
-  width: 30%;
+  align-items: center;
+  gap: 2%;
+  margin-bottom: 2%;
+  padding: 2% 4%;
+  border: 1px solid #ccc;
+  border-radius: 4px;
 }
 
-.form-container {
+.detail {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  width: 90%;
-  margin: 0 auto;
-  color: white;
+  align-items: flex-start;
+  width: 40%;
 }
 
-.form-container input {
-  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
-  font-size: 1.5rem;
-  margin-top: 2%;
-  margin-left: 4%;
-  width: 30%;
-  padding: 1%;
-  background-color: rgba(255, 255, 255, 0.5);
+.title {
+  align-self: flex-start;
+  font-family: "Roboto Slab", serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.line {
+  width: 0.055rem;
+  height: 4.375rem;
+  margin: 1.125rem 3.688rem 0 4.938rem;
+  border: solid 1px #b9b7b728;
+  background-color: #e8e8e8;
+}
```

Comment:
- [ ] Please consider changing the border color to `#e8e8e8` as present in the Zeplin design guidelines 💪😃